ci-operator/config/openshift/lws-operator/openshift-lws-operator-main.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  ocp_4.19_base-rhel9:
+    name: "4.19"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.23-openshift-4.19
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.ci
+  inputs:
+    ocp_4.19_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.19:base-rhel9
+    ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19
+  to: lws-operator
+promotion:
+  to:
+  - name: "4.19"
+    namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.19"
+  namespace: ocp
+tests:
+- as: verify
+  commands: make verify-gofmt
+  container:
+    from: src
+- as: unit
```

Comment:
I suppose, we will add e2es later as well, right?

Reply:
yes, that is correct.